src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,51 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+
+    if (this.health === 100) {
+      Animal.alive.push(this);
+    }
+  }
+
+  die() {
+    const index = Animal.alive.indexOf(this);
+
+    if (index > -1) {
+      Animal.alive.splice(index, 1);
```

Comment:
The `die` method removes the animal from the `alive` array, which is correct. However, ensure that this method is called whenever an animal's health drops to 0 or below, as it is currently only called in the `bite` method of `Carnivore`.